conanfile.txt

```diff
@@ -0,0 +1,10 @@
+[requires]
+nlohmann_json/3.11.2
+gegles-spdlog_setup/1.1.0
+cpptoml/0.1.1
+spdlog/1.13.0
+fmt/10.2.1
+
```

Comment:
```suggestion

[layout]
cmake_layout

```
Makes it easier to `conan install .`. Otherwise you need to provide additional arguments for the build dir location, etc.